src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/VB6RandomFile.vb

```diff
@@ -10,7 +11,7 @@ Imports Microsoft.VisualBasic.CompilerServices.Utils
 
 Namespace Microsoft.VisualBasic.CompilerServices
 
-    <System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)> _
+    <System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)>
```

Comment:
Intentional?

Reply:
Nope, VS keeps removing that for me though, had to re-add it back via notepad. Not sure what it is for or if it was just a typo, @cston any clues?

Reply:
These are just legacy line continuation characters. If VS removes the continuation characters, it should be fine to remove them.

Reply:
oh, I saw your reply after I had reverted the changes back, I can take care of them again if you want me to @cston or I can leave them as is for now.

Reply:
It's fine to leave the line continuation characters as is, or to remove them if VS removes them automatically, whichever is easier.